<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveVolumeGroupBox</class>
 <widget class="QGroupBox" name="SaveVolumeGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saving Volumes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Data folder</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
      <item>
       <widget class="QPushButton" name="pbSelectFolder">
        <property name="text">
         <string>Select Folder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelFolder">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="sbN">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbSaveFixedN">
         <property name="text">
          <string>Save Fixed N</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbSaveContinuous">
       <property name="text">
        <string>Save Continuous</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelStatus">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
